content/Hardware Support/Generic/Update-the-firmware-for-WiFiNINA-and-WiFi101.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Update the firmware for WiFiNINA and WiFi101"
+id: 360013896579
+---
+
+Learn how to update the firmware for WiFiNINA and WiFi101 using the Firmware Updater tool.
+
+In this article:
+
+* [Compatible Boards](#compatible-boards)
+* [Update the firmware in Arduino IDE 2](#update-the-firmware-in-arduino-ide-2)
+* [Update the firmware in Arduino IoT Cloud](#update-the-firmware-in-arduino-iot-cloud)
+
+---
+
+<a id="compatible-boards"></a>
+
+## Compatible Boards
+
+Boards with the **WiFiNINA** module:
+
+* [Arduino MKR WiFi 1010](https://store.arduino.cc/arduino-mkr-wifi-1010)
+* [Arduino MKR Vidor 4000](https://store.arduino.cc/arduino-mkr-vidor-4000)
+* [Arduino Nano 33 IoT](https://store.arduino.cc/arduino-nano-33-iot)
+* [Arduino UNO WiFi Rev2](https://store.arduino.cc/arduino-uno-wifi-rev2)
+* [Arduino Nano RP2040 Connect](https://store.arduino.cc/collections/boards/products/arduino-nano-rp2040-connect)
+
+Boards with the **WiFi101** module:
+
+* [Arduino MKR 1000 WiFi](https://docs.arduino.cc/hardware/mkr-1000-wifi)
```

Comment:
* It occurs to me that this article was confusing the library names (WiFi101 and WiFiNINA) with the names of the connectivity modules, which are different.
* Since we're dropping the instructions that involved opening an Example from the respective library, I don't think there's any need to emphesize this categorization—it can just be a single list of boards that can be used with the tool.

```suggestion
These boards uses the WiFi101 or WiFiNINA library:

* [Arduino MKR WiFi 1010](https://store.arduino.cc/arduino-mkr-wifi-1010)
* [Arduino MKR Vidor 4000](https://store.arduino.cc/arduino-mkr-vidor-4000)
* [Arduino Nano 33 IoT](https://store.arduino.cc/arduino-nano-33-iot)
* [Arduino UNO WiFi Rev2](https://store.arduino.cc/arduino-uno-wifi-rev2)
* [Arduino Nano RP2040 Connect](https://store.arduino.cc/collections/boards/products/arduino-nano-rp2040-connect)
* [Arduino MKR 1000 WiFi](https://docs.arduino.cc/hardware/mkr-1000-wifi)
```

Reply:
I agree. It also adds clarity to the article.